apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/success/en.ftl

```diff
@@ -1,8 +1,8 @@
-next-payment-confirmation-thanks-heading = Thank you!
+next-payment-confirmation-thanks-heading = Thanks, now check your email!
 
 # $email (String) - The user's email.
 # $product_name (String) - The name of the subscribed product.
-next-payment-confirmation-thanks-subheading = A confirmation email has been sent to { $email } with details on how to get started with { $product_name }.
+next-payment-confirmation-thanks-subheading = You'll receive an email at { $email } with instructions about your subscription, as well as your payment details.
```

Comment:
We can re-use this fluent ID with the `next-` prefix as well:
```
# $email (string) - The user's email.
payment-confirmation-thanks-subheading-account-exists = You’ll receive an email at { $email } with instructions for setting up your account, as well as your payment details.
```

Also, just so you're aware, we use [smart characters](https://platypiduses.com/dashes-quotes-ellipses/) (at least that's what I think they're called?) in ftl strings, eg:
- `’` instead of `'`
- `…` instead of `...`

Reply:
Thanks for the note about the smart characters! I am not sure I understand about the fluent id re-use though, the two strings are similar, but not the same.

Reply:
Aah, I didn't catch that the sentence changed at the end! Apologies, this would require a new fluent id.